Cargo.toml

```diff
@@ -19,6 +19,7 @@ ion-rs = "~0.6.0"
 libc = "0.2"
 memmap = "0.7.0"
 tempfile = "3.2.0"
+ion-schema-rust = { git = "https://github.com/amzn/ion-schema-rust" }
```

Comment:
We should publish the current version of ion-schema to crates.io so we can use a version number in the dependency.